src/CDSAPI.jl

```diff
@@ -2,6 +2,50 @@ module CDSAPI
 
 using HTTP
 using JSON
+using Base.ScopedValues
+
+const auth = ScopedValue(Dict("url" => "", "key" => ""))
+
+"""
+    CDScredentials()
```

Comment:
```suggestion
    credentials()
```

Please find/replace all occurrences of the old name.